README.md

```diff
@@ -80,28 +80,22 @@ make freeze-requirements
 
 ### Updating the index mapping
 
-The mapping JSON file is generated from framework data, and the originals reside alongside the framework data for the
-latest relevant framework iteration in the
-[frameworks repository](https://github.com/alphagov/digitalmarketplace-frameworks/).
+The index mapping is generated using the [`generate-search-config.py`] script
+from framework-specific mapping templates in the [frameworks repository].
 
-For example, the `mappings/services.json` file was generated by the
-[generate-search-config.py](https://github.com/alphagov/digitalmarketplace-frameworks/blob/master/scripts/generate-search-config.py)
-script from an original mapping file
-[frameworks/g-cloud-9/search_mapping.json](https://github.com/alphagov/digitalmarketplace-frameworks/blob/master/frameworks/g-cloud-9/search_mapping.json).
+For example, to generate the services mapping for G-cloud 10, you could run
```

Comment:
I'd do `generate` -> `(re-)generate` to make it clear that that is something that Sometimes Happens.